### PR TITLE
[SCB-790] integration test support create new target microservice instance in one node

### PR DESCRIPTION
**Pull request by wujimin**

### CODE REVIEW

File: demo/pom.xml [hunk 1]
```diff
@@ -181,6 +181,7 @@
       <plugin>
         <groupId>org.sonatype.plugins</groupId>
```
Comment (WillemJiang) [inline demo/pom.xml]: I think we don't need this staging maven plugin any more, as we don't deploy the artifacts into nexus public repo.

File: integration-tests/it-common-service/pom.xml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (WillemJiang) [inline integration-tests/it-common-service/pom.xml]: We need to add the Apache License header here.

File: pom.xml [hunk 3]
```diff
@@ -313,4 +227,101 @@
       </plugins>
     </pluginManagement>
   </build>
+
+  <profiles>
+    <profile>
```
Comment (WillemJiang) [inline pom.xml]: I think we need to add the deploy profile at the same time. 
You can find the travis deploy command [here](https://github.com/apache/incubator-servicecomb-java-chassis/blob/master/scripts/travis.sh#L59)

File: demo/pom.xml (hunk 1 above)
Comment (wujimin) [reply demo/pom.xml]: done

File: integration-tests/it-common-service/pom.xml (hunk 2 above)
Comment (wujimin) [reply integration-tests/it-common-service/pom.xml]: done

File: integration-tests/it-common/pom.xml [hunk 4]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
```
[the remark this answers is not in the document]
Comment (wujimin) [reply integration-tests/it-common/pom.xml]: done

File: integration-tests/it-consumer/pom.xml [hunk 5]
```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
```
[the remark this answers is not in the document]
Comment (wujimin) [reply integration-tests/it-consumer/pom.xml]: done

File: integration-tests/it-consumer/src/main/java/org/apache/servicecomb/it/testcase/base/TestDownload.java [hunk 6]
```diff
@@ -0,0 +1,142 @@
...
+//    intf.tempFileEntity("abc");
+  }
+
+  @Test
+  public void test2() {
+    x++;
+    System.out.println("test2");
+    Assert.assertEquals(x, x + 1);
+  }
+//  private File dir = new File("target/download");
```
[the remark this answers is not in the document]
Comment (wujimin) [reply integration-tests/it-consumer/src/main/java/org/apache/servicecomb/it/testcase/base/TestDownload.java]: commented code will be used in the future.

File: integration-tests/it-edge/pom.xml [hunk 7]
```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
```
[the remark this answers is not in the document]
Comment (wujimin) [reply integration-tests/it-edge/pom.xml]: done

File: integration-tests/it-producer/pom.xml [hunk 8]
```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
```
[the remark this answers is not in the document]
Comment (wujimin) [reply integration-tests/it-producer/pom.xml]: done

File: pom.xml (hunk 3 above)
Comment (wujimin) [reply pom.xml]: there is no any log for maven-deploy-plugin

**Pull request merged**